content/_data/changelogs/lts.yml

```diff
@@ -593,7 +593,8 @@
   lts_predecessor: "2.60.3"
   lts_baseline: "2.73"
   banner: >
-    2.73.1 has been released with a known issue involving agent connections.
+    Two regressions since the previous LTS release have been identified in 2.73.1.
+    One involved remoting (agent) connections and the other concerns failures to use password-protected SSH keys.
```

Comment:
Would you be willing to change

> password-protected SSH keys

into

> ed25519 passphrase protected private keys

The text is not much longer, and it will reduce unnecessary panic from people who are not yet using ed25519 private keys.

Even if you don't update the text to include ed25519, I would prefer "passphrase" rather than "password" if possible, since "passphrase" seems to be the word used by the tools which create private keys.